ggshield/cmd/honeytoken/create.py

```diff
@@ -80,7 +80,11 @@ def create_cmd(
     if not isinstance(response, (Detail, HoneytokenResponse)):
         raise UnexpectedError("Unexpected honeytoken response")
 
-    if isinstance(response, Detail) and response.status_code == 403:
+    if (
```

Comment:
We should probably enrich [this test](https://github.com/GitGuardian/ggshield/blob/main/tests/unit/cmd/honeytoken/test_honeytoken.py) WDYT ?